pkgs/development/interpreters/lua-5/wrapper.nix

```diff
@@ -0,0 +1,70 @@
+{ stdenv, lua, buildEnv, makeWrapper
+, extraLibs ? []
+, postBuild ? ""
+, ignoreCollisions ? false
+, lib
+, requiredLuaModules
+}:
+
+# Create a lua executable that knows about additional packages.
+let
+  env = let
+    paths =  requiredLuaModules (extraLibs ++ [ lua ] );
+  in buildEnv {
+    name = "${lua.name}-env";
+
+    inherit paths;
+    inherit ignoreCollisions;
+
+    # we create wrapper for the binaries in the different packages
+    postBuild = ''
+
+      . "${makeWrapper}/nix-support/setup-hook"
+
+      # get access to lua functions
+      . ${lua}/nix-support/setup-hook
+
+      if [ -L "$out/bin" ]; then
+          unlink "$out/bin"
```

Comment:
How is the semantic different from using `rm` here?

Reply:
copy pasted from https://github.com/NixOS/nixpkgs/blob/222f905fd9042f80d4cdef65bb7d2801fdb63429/pkgs/development/interpreters/python/wrapper.nix#L23. According to man, the condition is entered only if $out/bin is a symlink. Not sure we need that.

Reply:
@FRidh when does the case triggers for python projects?

Reply:
f6e835a2c3e6e56aea843164c93d81716fc20ac0

Reply:
@teto I don't think we have that problem yet in Lua.